nrfx/samples/doc/buildjars.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
`buildfiles_ic` directory, `manifest` directory, `nrfx_examples_ic.doxyfile` file and `buildjars.xml` file should be removed